Lib/random.py

```diff
@@ -48,25 +48,13 @@
 from warnings import warn as _warn
 from math import log as _log, exp as _exp, pi as _pi, e as _e, ceil as _ceil
 from math import sqrt as _sqrt, acos as _acos, cos as _cos, sin as _sin
-from math import tau as TWOPI, floor as _floor
-try:
-    from os import urandom as _urandom
-except ImportError:
-    # XXX RUSTPYTHON
-    # On wasm, _random.Random.random() does give a proper random value, but
-    # we don't have the os module
-    def _urandom(*args, **kwargs):
-        raise NotImplementedError("urandom")
-    _os = None
+from math import tau as TWOPI, floor as _floor, isfinite as _isfinite
+from os import urandom as _urandom
 from _collections_abc import Set as _Set, Sequence as _Sequence
+from operator import index as _index
 from itertools import accumulate as _accumulate, repeat as _repeat
 from bisect import bisect as _bisect
-try:
-    import os as _os
-except ImportError:
-    # XXX RUSTPYTHON
-    # On wasm, we don't have the os module
-    _os = None
```

Comment:
Can you make sure to keep these try blocks? They keep `random` working on wasm.

Reply:
@Snowapril could you check this comment?

Reply:
Oh, I'm sorry noticed so late 😭. I rebased it.